Utility/SQL/Migration.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python
+# coding: utf-8
+
+import argparse
+from subprocess import Popen, PIPE
+import psycopg2
+import os.path
+
+PRODUCT_DATABASE_NAME = os.getenv('DATABASE_NAME')
+
+kill_db_processes_sql = """SELECT    pg_terminate_backend (pid)
+                        FROM    pg_stat_activity
+                        WHERE   datname = '{}';
+                    """.format(PRODUCT_DATABASE_NAME)
+
+new_database_name_sql = """ALTER DATABASE {0} RENAME TO _{0};
+                """.format(PRODUCT_DATABASE_NAME)
+
+
+def connection_parameters(db_name_env='POSTGRES_DB') -> str:
+    _args = "password=%s " % os.getenv('POSTGRES_PASSWORD') + \
+           "host=%s " % os.getenv('POSTGRES_HOST') + \
+           "port=%s " % os.getenv('POSTGRES_PORT') + \
+           "dbname=%s " % os.getenv(db_name_env) + \
+           "user=%s " % os.getenv('POSTGRES_USER')
+    return _args
+
+
+def grab_sql_request() -> str:
+    path = "./"
+    if os.path.isfile('Database_Scheme.sql'):
+        path += 'Database_Scheme.sql'
+    else:
+        raise RuntimeError('Failed to open Database_Scheme.sql')
+    request = None
+    with open(path, "r") as source:
+        request = source.read()
+    return request
+
+
+def make_table_copy() -> None:
+    conn = None
+    try:
+        conn = psycopg2.connect(connection_parameters())
+        cur = conn.cursor()
+        cur.execute(kill_db_processes_sql)
+        cur.execute(new_database_name_sql)
```

Comment:
I don't pretend to be an expert here, but I think you need to commit every operation with the database.

Reply:
Indeed! You are not at expert here!
https://www.psycopg.org/docs/usage.html

Reply:
This is just my opinion :)